packages/contracts/contracts/test/MajorityVotingMock.sol

```diff
@@ -5,38 +5,26 @@ pragma solidity 0.8.10;
 import "../voting/majority/MajorityVotingBase.sol";
 
 contract MajorityVotingMock is MajorityVotingBase {
-    function initializeMock(
-        IDAO _dao,
-        uint64 _supportThreshold,
-        uint64 _minParticipation,
-        uint64 _minDuration,
-        uint256 _minProposerVotingPower
-    ) public initializer {
-        __MajorityVotingBase_init(
-            _dao,
-            _supportThreshold,
-            _minParticipation,
-            _minDuration,
-            _minProposerVotingPower
-        );
+    function initializeMock(IDAO _dao, PluginSettings calldata _pluginSettings) public initializer {
+        __MajorityVotingBase_init(_dao, _pluginSettings);
     }
 
     function createProposal(
         bytes calldata, /* _proposalMetadata */
         IDAO.Action[] calldata, /* _actions */
         uint64, /* _startDate */
         uint64, /* _endDate */
-        bool, /* _executeIfDecided */
-        VoteOption /* _choice */
+        bool, /* _tryEarlyExecution */
+        VoteOption /* _voteOption */
```

Comment:
do we still allow voting upon creation? Is this a significant use case enough?

Reply:
Yes, we still support it.
I think that this feature was introduced/requested by Ramon based on user research. Maybe @novaknole has more info

Reply:
Correct @Michael-A-Heuer .. This was a requested feature. Most likely, when you create a proposal, you directly want to put your vote right away to avoid 2 separate tx.

Reply:
After a short chat with @sioeth, this flow will be not be supported on the frontend for now, but can be in the future.